### PR TITLE
Cert

### PR DESCRIPTION
**Pull request by nzoschke**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM convox/alpine:3.1
+FROM gliderlabs/alpine:edge
```
Comment (nzoschke) [inline Dockerfile]: Is it ok to check this in? We require HAProxy 1.5 for SSL support. @ddollar

File: bin/gen-cert [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+mkdir -p /etc/ssl/convox
+
+openssl genrsa -des3 -passout pass:x -out /etc/ssl/convox/server.pass.key 2048
+openssl rsa -passin pass:x -in /etc/ssl/convox/server.pass.key -out /etc/ssl/convox/server.key
+
+rm /etc/ssl/convox/server.pass.key
+
+openssl req -new -key /etc/ssl/convox/server.key -out /etc/ssl/convox/server.csr \
+  -subj "/C=US/ST=California/L=San Francisco/O=Convox/OU=Kernel/CN=*.elb.amazonaws.com"
```
Comment (nzoschke) [inline bin/gen-cert]: Not sure what the best CN to use is. This doesn't seem to help Chrome.

**Comment by nzoschke** [comment]
Should I commit e81ff170c33b0e7e838a09d0b546662d9cd9c51e ?

File: Dockerfile (hunk 1 above)
Comment (ddollar) [inline Dockerfile]: Would prefer to use the haproxy-1.5 package for 3.1 that I gave you so we're slightly less bleeding-edge.

File: bin/gen-cert (hunk 2 above)
Comment (ddollar) [inline bin/gen-cert]: What does the error look like in Chrome?

**Comment by ddollar** [comment]
Yes

**Comment by nzoschke** [comment]
I get this, even when serving from an ELB. I would have hoped something could make at least the "Server's certificate does not match the URL." error go away, but this is not yet it.

screen shot 2015-05-26 at 12 04 43 pm

**Comment by ddollar** [comment]
:+1:

**Comment by nzoschke** [comment]
I think the SSL errors are ok for now. You can ignore it with a couple of clicks, and anyone serious should configure a real cert.

There could be another pass about making a convox authority that people can install on the system, but that's a big scope increase.

**Pull request merged**